config/css.js

```diff
@@ -41,6 +40,6 @@ gulp.task('css:prefix-release', function () {
     .pipe(gulp.dest('./css/'));
 });
 
-gulp.task('css:release', function (done) {
```

Comment:
I noticed the `css/` folder with the version exapanded and minified version of `rivet.css` weren't getting built into the root of the directory when running `npm run build`. Which also means that there's no `css` folder added to the final `rivet.zip` that gets built.

You can get rid of this callback function here (`function(done)`) and it will fix it. So it would become:

```javascript
gulp.task('css:release', gulp.series(
  'css:dist',
  'css:prefix-release',
  'css:header',
  'css:minify'
));
```
Just like you have the javascript `js:release` task set up below.